packages/react-components/virtualizer/src/useVirtualizer.ts

```diff
@@ -0,0 +1,403 @@
+import { useIntersectionObserver } from './useIntersectionObserver';
+import type { ReactNode } from 'react';
+import { useMemo, useRef, useState } from 'react';
+import * as React from 'react';
+
+import type { VirtualizerProps, VirtualizerState } from './Virtualizer.types';
+import { resolveShorthand } from '@fluentui/react-utilities';
+
+export function useVirtualizer_unstable(props: React.PropsWithChildren<VirtualizerProps>): VirtualizerState {
+  const {
+    itemSize,
+    virtualizerLength,
+    children,
+    sizeOfChild,
+    bufferItems = Math.round(virtualizerLength / 4.0),
+    bufferSize = Math.floor(bufferItems / 2.0) * itemSize,
+    scrollViewRef,
+    isReversed = false,
+    isHorizontal = false,
+    onUpdateIndex,
+    onCalculateIndex,
+  } = props;
+
+  // Safe access array version of children
+  const childArray = useMemo(() => (Array.isArray(children) ? children : [children]), [children]);
```

Comment:
https://reactjs.org/docs/react-api.html#reactchildrentoarray

React already has a utility for this
```suggestion
  const childArray = React.Children.toArray(children)
```